src/OpenLoco/Windows/TileInspector.cpp

```diff
@@ -0,0 +1,498 @@
+                    auto object = ObjectManager::get<road_object>(objectId);
+                    return object->name;
+                }
+                break;
+            }
+            case element_type::industry:
+            {
+                auto industry = element.asIndustry();
+                if (industry != nullptr)
+                {
+                    auto object = ObjectManager::get<industry_object>(industry->industry()->object_id);
+                    return object->name;
+                }
+            }
+        }
+        return StringIds::empty;
+    }
+
+    static string_id getOwnerName(const tile_element& element)
+    {
+        if (element.type() == element_type::road)
+        {
+            auto road = element.asRoad();
+            if (road != nullptr)
+            {
+                auto ownerId = road->owner();
+                if (ownerId != CompanyId::neutral)
+                {
+                    auto company = CompanyManager::get(ownerId);
+                    return company->name;
+                }
+            }
+        }
+        else if (element.type() == element_type::track)
+        {
+            auto track = element.asTrack();
+            if (track != nullptr)
+            {
+                auto ownerId = track->owner();
+                if (ownerId != CompanyId::neutral)
+                {
+                    auto company = CompanyManager::get(ownerId);
+                    return company->name;
+                }
+            }
+        }
+        return StringIds::empty;
+    }
+
+    static void drawScroll(Ui::window* self, Gfx::drawpixelinfo_t* const context, uint32_t)
+    {
+        if (_currentPosition == map_pos(0, 0))
+            return;
+
+        auto tile = TileManager::get(_currentPosition);
+        auto yPos = 0;
+        auto rowNum = 0;
+        for (auto& element : tile)
+        {
+            string_id formatString;
+            if (self->var_842 == rowNum)
+            {
+                Gfx::fillRect(context, 0, yPos, self->width, yPos + self->row_height, Colour::aquamarine);
+                formatString = StringIds::white_stringid;
+            }
+            else if (self->row_hover == rowNum)
+            {
+                Gfx::fillRect(context, 0, yPos, self->width, yPos + self->row_height, 0x2000030);
+                formatString = StringIds::wcolour2_stringid;
+            }
+            else
+            {
+                formatString = StringIds::wcolour2_stringid;
+            }
+
+            FormatArguments args = {};
+
+            string_id elementName = getElementTypeName(element);
+            string_id objectName = getObjectName(element);
+            string_id ownerName = getOwnerName(element);
+
+            if (ownerName != StringIds::empty)
+            {
+                args.push(StringIds::tile_inspector_entry_three_pos);
+                args.push(objectName);
+                args.push(ownerName);
+                args.push(elementName);
```

Comment:
The element name does not show for railroad track pieces, for some reason; it's just left blank. I don't understand why not.

Reply:
@duncanspumpkin Any ideas what may be happening here?